src/main/java/org/alliancegenome/curation_api/model/ingest/dto/IngestDTO.java

```diff
@@ -12,6 +12,10 @@
 @Getter
 public class IngestDTO {
 	
+	@JsonView({View.FieldsOnly.class})
+	@JsonProperty("linkml_version")
```

Comment:
I think this is going to cause push back... but I guess we will see.

Reply:
I'm hoping not.  The DQMs are used to submitting metadata, and personally I find it useful to be easily identify the schema version submitted to by looking at the file.